source/Engine.cpp

```diff
@@ -1568,11 +1568,27 @@ void Engine::CalculateStep()
 		DoScanning(it);
 
 	// Draw the objects. Start by figuring out where the view should be centered:
-	Point newCenter = center;
+	newCenter = center;
 	Point newCenterVelocity;
 	if(flagship)
 	{
+		// Code block to calculate the center of the viewport.
+		// If Camera Acceleration is enabled an offset calculated through the
+		// flagship's velocity will be added to the center.
 		newCenter = flagship->Center();
+		double prefMul = Preferences::CameraAcceleration() == "on" ? 1.
+			: (Preferences::CameraAcceleration() == "reversed" ? -1. : 0.);
```

Comment:
```suggestion
		const string &pref = Preferences::CameraAcceleration();
		double prefMul = pref  == "on" ? 1. : (pref  == "reversed" ? -1. : 0.);
```
Thoughts on renaming `prefMul` to `accelDirection`/`offsetDirection`?